Disable the MD5 build sum check that breaks all out-of-tree plugins.
It's not a change it should be done on a stable release, anyway...

Leave the field in the struct and the macro that generates it however so
that we can be both backward and forward API/ABI compatible.

 -- Faidon Liambotis <paravoid@debian.org>

--- a/main/loader.c
+++ b/main/loader.c
@@ -76,8 +76,6 @@ static unsigned char expected_key[] =
 { 0x87, 0x76, 0x79, 0x35, 0x23, 0xea, 0x3a, 0xd3,
   0x25, 0x2a, 0xbb, 0x35, 0x87, 0xe4, 0x22, 0x24 };
 
-static char buildopt_sum[33] = AST_BUILDOPT_SUM;
-
 static unsigned int embedding = 1; /* we always start out by registering embedded modules,
 				      since they are here before we dlopen() any
 				   */
@@ -615,13 +613,6 @@ static unsigned int inspect_module(const
 		return 1;
 	}
 
-	if (!ast_strlen_zero(mod->info->buildopt_sum) &&
-	    strcmp(buildopt_sum, mod->info->buildopt_sum)) {
-		ast_log(LOG_WARNING, "Module '%s' was not compiled with the same compile-time options as this version of Asterisk.\n", mod->resource);
-		ast_log(LOG_WARNING, "Module '%s' will not be initialized as it may cause instability.\n", mod->resource);
-		return 1;
-	}
-
 	return 0;
 }
 
